Assets/Scripts/Game/PlayerActivate.cs

```diff
@@ -502,70 +503,92 @@ void Update()
                     return;
                 }
 
+                // Play sound when bashing.
+                if (isBash && door.doorType != DoorTypes.DungeonExit)
+                    if (TryGetComponent<DaggerfallAudioSource>(out var dfAudioSource))
+                        dfAudioSource.PlayOneShot(SoundClips.PlayerDoorBash);
+
                 if (door.doorType == DoorTypes.Building && !playerEnterExit.IsPlayerInside)
                 {
                     // Discover building
                     GameManager.Instance.PlayerGPS.DiscoverBuilding(building.buildingKey);
 
                     // Handle clicking exterior door with Open spell active
-                    if (HandleOpenEffectOnExteriorDoor(buildingLockValue))
+                    if (HandleOpenEffectOnExteriorDoor(buildingLockValue) && !isBash)
```

Comment:
I think you put the check in the wrong order. Currently, a bash will consume an incumbent Open effect, but will not activate it. If you rewrite it as `!isBash && HandleOpenEffectOnExteriorDoor(buildingLockValue)`, then a bash will neither consume nor activate the Open effect.

Reply:
I've also noticed another DFU issue here: by making the door "unlocked", you enable the "greeting" message, despite having bypassed the lock. This is inconsistent with lockpicking, which moves forward down the function without unlocking the door.

The check should probably be rewritten as `if(!buildingUnlocked || !HandleOpenEffectOnExterior(buildingLockValue)` (assuming you still have the `!isBash` check above), so that `buildingUnlocked` remains false, and the greeting message will be avoided like in the lockpicking case.